admin-dev/themes/default/js/bundle/module/module_card.js

```diff
@@ -144,10 +144,9 @@ var AdminModuleCard = function () {
                     var alteredSelector = null;
                     var mainElement = null;
                     if (action == "uninstall") {
-                        jqElementObj.html("");
-                        jqElementObj.fadeOut(function() {
-                            $(this).remove();
-                        });
+                        alteredSelector = _this.getModuleItemSelector().replace('.', '');
+                        mainElement = jqElementObj.parents('.' + alteredSelector).first();
+                        mainElement.remove();
```

Comment:
Some comments from users of the module page say merchants do not see immediately when a module is modified. Can't we leave the fadeOut effet ?